src/main/scala/beam/agentsim/agents/ridehail/RideHailModifyPassengerScheduleManager.scala

```diff
@@ -418,6 +418,7 @@ class RideHailModifyPassengerScheduleManager(
 
   def isVehicleNeitherRepositioningNorProcessingReservation(vehicleId: Id[Vehicle]): Boolean = {
     // TODO: FIXME `vehicleIdToModifyPassengerScheduleStatus` is broken, so for now we return `true`, but fixme, please!
+    log.warning(s"`vehicleIdToModifyPassengerScheduleStatus` is broken and variable vehicleId($vehicleId) is not used")
```

Comment:
Should we instead/also create a github issue?

Reply:
Issue created: https://github.com/LBNL-UCB-STI/beam/issues/3296